sender.go

```diff
@@ -177,21 +177,34 @@ func (w *Sender) sendMessage(msg Msg) {
 
 	// failing on a lookup isn't a halting problem but we should log it
 	if err != nil {
-		msgLog.WithError(err).Warning("error looking up msg was sent")
+		log.WithError(err).Warning("error looking up msg was sent")
+	}
+
+	// is this msg in a loop?
+	loop, err := backend.IsMsgLoop(sendCTX, msg)
+
+	// failing on loop lookup isn't permanent, but log
+	if err != nil {
+		log.WithError(err).Warning("error looking up msg loop")
```

Comment:
if this is just a warning, any chance it stops working and we never notice?

Reply:
I *think* warnings still show up in Sentry, but actually I do prefer these being real errors from a sentry perspective, will change.